airbyte-cdk/python/airbyte_cdk/sources/streams/http/requests_native_auth/abstract_oauth.py

```diff
@@ -80,7 +80,7 @@ def build_refresh_request_body(self) -> Mapping[str, Any]:
             "refresh_token": self.get_refresh_token(),
         }
 
-        if self.get_scopes:
+        if self.get_scopes():
```

Comment:
@brianjlai flagging this because this is a change in behavior. The existing condition always evaluates to true since the method always exists. I'm pretty sure this is a bug but I don't know how to verify the new behavior works for all possible connectors (including connector-builder connectors)...

Would it make more sense to remove the condition to keep the existing behavior but make the code clearer?

Reply:
So basically this would mean retaining the existing behavior means that we can set a token with a payload where scopes = `[]`?

I'd be willing to take the risk and say we only set the scopes payload field if theres at least one to pass along. So how you have it seems fine and was the original intent of the code